src/StripeTests/Services/AutoPagingTest.cs

```diff
@@ -0,0 +1,107 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Moq;
+    using Moq.Protected;
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class AutoPagingTest : BaseStripeTest
```

Comment:
This test seems pretty complex, but it's actually pretty straightforwards:
1. set up 3 stubbed requests with file fixtures
2. call `ListAutoPaging`
3. check invocations to make sure the correct parameters were sent